app/models/concerns/simple_state_machine.rb

```diff
@@ -31,6 +31,13 @@ def state_machine_config(column: :state, &block)
         scope state_name, -> { where(ssm_column => state_name.to_s) }
       end
 
+      # Add validation for state column
+      validates ssm_column, inclusion: {
+        in: state_machine_states.map(&:to_s),
+        message: "must be one of: #{state_machine_states.map(&:to_s).join(', ')}",
+        allow_nil: true,
```

Comment:
In practice all the classes the use this have a non-nullable column, but I figured the Concern itself shouldn't make that assumption

Reply:
This is fine as is. It's academic since we don't have nullable status but it feels more intuitive to me to exclude nulls; a null status isn't really valid so it seems like a safe assumption to make 🤷